pkg/controller/compliance/compliance_controller.go

```diff
@@ -364,7 +365,16 @@ func (r *ReconcileCompliance) Reconcile(ctx context.Context, request reconcile.R
 		r.status.SetDegraded(operatorv1.ResourceNotReady, "Elasticsearch gateway certificates are not available yet, waiting until they become available", nil, reqLogger)
 		return reconcile.Result{}, nil
 	}
-	linseedCertificate, err := certificateManager.GetCertificate(r.client, render.TigeraLinseedSecret, common.OperatorNamespace())
+
+	// The location of the Linseed certificate varies based on if this is a managed cluster or not.
+	// For standalone and management clusters, we just use Linseed's actual certificate.
+	linseedCertLocation := render.TigeraLinseedSecret
+	if managementClusterConnection != nil {
+		// For managed clusters, we need to add the certificate of the Voltron endpoint. This certificate is copied from the
+		// management cluster into the managed cluster by kube-controllers.
+		linseedCertLocation = render.VoltronLinseedPublicCert
```

Comment:
I will probably need some similar changes to DPI.

Reply:
Yep :+1: